docs/topics/runners/gradle.md

```diff
@@ -0,0 +1,539 @@
+[//]: # (title: Gradle plugin)
+
+To generate documentation for a Gradle-based project, you can use [Dokka Gradle plugin](#applying-the-plugin).
+
+It comes with basic autoconfiguration (including multimodule and multiplatform projects), has convenient 
+[Gradle tasks](#generating-documentation) for generating documentation, and provides a great deal of
+[configuration options](#configuration) to customize output.
+
+## Applying the plugin
+
+The recommended way of applying the plugin is via 
+[plugins DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block):
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+plugins {
+    id("org.jetbrains.dokka") version "%dokkaVersion%"
+}
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+plugins {
+    id 'org.jetbrains.dokka' version '%dokkaVersion%'
+}
+```
+
+</tab>
+</tabs>
+
+> Under the hood, Dokka uses [Kotlin Gradle plugin](https://kotlinlang.org/docs/gradle.html) to perform autoconfiguration
+> of [source sets](https://kotlinlang.org/docs/multiplatform-discover-project.html#source-sets) for which documentation
+> should be generated, so make sure Kotlin Gradle Plugin is applied as well, or configure source sets manually.
+>
+{type="note"} 
+
+> If you are using Dokka in a 
+> [precompiled script plugin](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:precompiled_plugins), 
+> you will have to add [Kotlin Gradle plugin](https://kotlinlang.org/docs/gradle.html) as a dependency in order for
+> it to work properly:
+>
+> <tabs group="build-script">
+> <tab title="Kotlin" group-key="kotlin">
+> 
+> ```kotlin
+> implementation(kotlin("gradle-plugin", "%kotlinVersion%"))
+> ```
+> 
+> </tab>
+> <tab title="Groovy" group-key="groovy">
+> 
+> ```groovy
+> implementation 'org.jetbrains.kotlin:kotlin-gradle-plugin:%kotlinVersion%'
+> ```
+> 
+> </tab>
+> </tabs>
+>
+{type="note"}
+
+
+### Legacy plugin application
```

Comment:
I believe we can omit this part, even for Kotlin Gradle plugin we don't write this. Users, who cannot use plugins DSL, probably, already know how to use this other method

Reply:
I like this idea very much, sir. Will fix